tests/main/presenters/test_search_presenters.py

```diff
@@ -302,21 +315,26 @@ def test_filter_groups_have_correct_state_when_changed(self):
         })
 
         search_filters = list(filters_for_lot('paas', g6_builder).values())
-        set_filter_states(search_filters, request)
+        lots_by_slug = get_lots_by_slug(self._get_framework_fixture_data('g-cloud-6')['frameworks'])
+        clean_request_query_params = clean_request_args(request.args, search_filters, lots_by_slug)
```

Comment:
They are very similar. I started out with them together as a parameterized test and I felt it was going to get too confusing, so I split them. In tests, I don't mind a little duplication if it makes them clearer and simpler.